mantidproject/mantid
# Disable throwing exception

**Description of work.**
We do not catch this exception, so there is no reason to throw it. Uncaught exception results in a segmentation fault.

This is a temporary solution, until we see how we can create the help window.


**Report to:** [user name]/[nobody]. 

**To test:**
Build workbench. Try to see in any algorithm the help button. It will show an error, but it will not crash

Fixes  #23978. 


*This does not require release notes* because **workbench**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
